onnx/defs/tensor/old.cc

```diff
@@ -426,40 +426,63 @@ ONNX_OPERATOR_SET_SCHEMA(
             "Constrain input 'X' and output 'Y' to all tensor types.")
         .SetDoc(Upsample_ver9_doc)
         .TypeAndShapeInferenceFunction([](InferenceContext& ctx) {
-          if (!hasNInputShapes(ctx, 1)) {
+          if (!hasNInputShapes(ctx, 2)) {
```

Comment:
same comment as previously - I think first input shape is good enough to proceed further....